app/bolt_listeners.py

```diff
@@ -411,826 +424,21 @@ def respond_to_new_message(
-            {
-                "type": "input",
-                "block_id": "where-to-share-summary",
-                "label": {
-                    "type": "plain_text",
-                    "text": "How would you like to see the summary?",
-                },
-                "element": {
-                    "action_id": "input",
-                    "type": "radio_buttons",
-                    "initial_option": where_to_display_options[0],
-                    "options": where_to_display_options,
-                },
-            },
-            {
-                "type": "input",
-                "block_id": "prompt",
-                "element": {
-                    "type": "plain_text_input",
-                    "action_id": "input",
-                    "multiline": True,
-                    "initial_value": prompt,
-                },
-                "label": {
-                    "type": "plain_text",
-                    "text": "Customize the prompt as you prefer:",
-                },
-            },
-            {
-                "type": "context",
-                "elements": [
-                    {
-                        "type": "mrkdwn",
-                        "text": "Note that after the instruction you provide, this app will append all the replies in the thread.",
-                    }
-                ],
-            },
-        ]
-    except SlackApiError as e:
-        is_error = True
-        error_code = e.response["error"]
-        if error_code == "not_in_channel":
-            blocks = [
-                {
-                    "type": "section",
-                    "text": {
-                        "type": "mrkdwn",
-                        "text": "It appears that this app's bot user is not a member of the specified channel. "
-                        f"Could you please invite <@{context.bot_user_id}> to <#{context.channel_id}> "
-                        "to make this app functional?",
-                    },
-                }
-            ]
-        else:
-            blocks = [
-                {
-                    "type": "section",
-                    "text": {
-                        "type": "mrkdwn",
-                        "text": f"Something is wrong! (error: {error_code})",
-                    },
-                }
-            ]
-
-    view = {
-        "type": "modal",
-        "callback_id": "request-thread-summary",
-        "title": {"type": "plain_text", "text": "Summarize the thread"},
-        "submit": {"type": "plain_text", "text": "Summarize"},
-        "close": {"type": "plain_text", "text": "Close"},
-        "private_metadata": json.dumps(
-            {
-                "thread_ts": thread_ts,
-                "channel": context.channel_id,
-            }
-        ),
-        "blocks": blocks,
-    }
-    if is_error is True:
-        del view["submit"]
-
-    client.views_open(
-        trigger_id=body.get("trigger_id"),
-        view=view,
-    )
-    ack()
-
```

Comment:
This is one example of a function (a shortcut) that we won't use, so it makes no sense to keep it in the listeners for now